a3p-integration/proposals/p:upgrade-19/package.json

```diff
@@ -27,7 +27,18 @@
     ]
   },
   "scripts": {
-    "agops": "yarn --cwd /usr/src/agoric-sdk/ --silent agops"
+    "agops": "yarn --cwd /usr/src/agoric-sdk/ --silent agops",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error 'lint:*'",
+    "lint:types": "tsc",
+    "lint:eslint": "eslint .",
```

Comment:
please don't perpetuate that every package gets its own lint commands. `a3p-integration` has a "lint" that includes all the proposals. the proposals are each a package but lightweight. 

It shouldn't have any scripts. (`postinstall` will be gone using Yarn 4 and the `agops` one will go away once we have signing in JS so we don't have to shell out for that)

Reply:
Whatever linting exists at `a3p-integration` is incomplete, because some typing issues failed to surface until I added these package-level scripts. Do you know how to address that gap?

Reply:
Partially... linting z:acceptance in #10499 caught a timer value that was implicitly assumed to be [seconds since the POSIX epoch](https://github.com/Agoric/agoric-sdk/pull/10499/files#diff-84e23aec574e7b15e52909e350ac3ad5124493641dd8e74e10bff307e2e6aaae), and investigation revealed [the flawed `waitUntil`](https://github.com/Agoric/agoric-sdk/pull/10499/files#diff-7419f6129439ae26183b76e622a166640a43c4c1b8a1c066c95505850622a8e1) that calculated a duration in the wrong direction and with the wrong units, which wasn't itself a typing issue but would have been much harder to find if such an issue were not adjacent.

Reply:
OK, I've consolidated linting at the a3p-integration/package.json level, including `tsc` for all proposal packages.